rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -120,11 +120,13 @@ def _insert_order(self, participant, order_id, tests, order_time, finalized_test
           finalized=finalized_time))
     return self.order_dao.insert(order)
 
-  def _insert_samples(self, participant, tests, sample_ids, confirmed_time, created_time):
+  def _insert_samples(self, participant, tests, sample_ids, identifier, confirmed_time,
```

Comment:
We should add a bit of test logic in here to confirm that sample order identifiers are actually being set properly.

Reply:
Ping?

Reply:
addressed below.